certified-connectors/Converter by Power2Apps/apiDefinition.swagger.json

```diff
@@ -3,676 +3,2194 @@
+          "x-ms-visibility": "important",
+          "x-ms-summary": "First match",
+          "description": ""
+        },
+        "firstMatchValue": {
+          "type": "string",
+          "x-ms-visibility": "important",
+          "x-ms-summary": "First match value",
+          "description": ""
+        }
+      }
+    },
+    "RegExpRequest": {
+      "required": [
+        "input"
+      ],
+      "type": "object",
+      "properties": {
+        "input": {
+          "minLength": 1,
+          "type": "string",
+          "x-ms-visibility": "important",
+          "x-ms-summary": "Text to match",
+          "description": "The string to search for a match"
+        },
+        "pattern": {
+          "type": "string",
+          "x-ms-visibility": "important",
+          "x-ms-summary": "Regular Expression",
+          "description": "The .NET regular expression pattern to match"
+        },
+        "option": {
+          "type": "string",
+          "x-ms-visibility": "important",
+          "x-ms-summary": "Regular expression option",
+          "description": "Select regular expression option",
+          "x-ms-dynamic-values": {
+            "operationId": "GetRegexOptions",
+            "value-path": "name",
+            "value-title": "name",
+            "parameters": {}
           }
-        ],
-        "operationId": "CreateWordFile",
-        "description": "Create a Word (DOCX) file",
-        "x-ms-visibility": "important"
+        }
       }
-    }
-  },
-  "definitions": {
-    "SectionArray": {
+    },
+    "Section": {
       "type": "object",
       "properties": {
         "sectionType": {
-          "x-ms-summary": "type",
+          "default": "Heading1",
+          "type": "string",
+          "x-ms-visibility": "important",
+          "x-ms-summary": "Type",
           "x-ms-dynamic-values": {
             "operationId": "GetWordSectionTypes",
             "value-path": "name",
-            "value-title": "name"
-          },
-          "x-ms-visibility": "important",
-          "type": "string"
+            "value-title": "name",
+            "parameters": {}
+          }
         },
         "sectionContent": {
           "type": "string",
           "x-ms-visibility": "important",
-          "x-ms-summary": "Content",
-          "description": "The content/text of the section"
+          "x-ms-summary": "Text",
+          "description": "Text"
+        }
+      }
+    },
+    "SharePointSearchResultResponse": {
+      "type": "object",
+      "properties": {
+        "title": {
+          "type": "string",
+          "x-ms-visibility": "important",
+          "x-ms-summary": "Title",
+          "description": "Title"
+        },
+        "link": {
+          "type": "string",
+          "x-ms-visibility": "important",
+          "x-ms-summary": "Link",
+          "description": "Link"
+        }
+      }
+    },
+    "SharepointResultData": {
+      "required": [
+        "sharepointResult"
```

Comment:
@stsiek The required input field has been added. Existing flows will not send this new field.